### PR TITLE
VMT: add KCSAs for CVE-2020-2024 and CVE-2020-2025

### PR DESCRIPTION
**Pull request by bergwolf**
We have fixed them and put the fix in a release. Let's
make the announcement.

/cc @YuvalAvra

### CODE REVIEW

File: VMT/KCSA/KCSA-CVE-2020-2025.md [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+
+  - name: Yuval Avrahami
+    affiliation: Yuval Avrahami
+    reported:
+      - CVE-2020-2025
+
+issues:
+
+  links:
+    - https://github.com/kata-containers/runtime/issues/2488
```
Comment (jodh-intel) [inline VMT/KCSA/KCSA-CVE-2020-2025.md:28]: Is this URL correct?

**Comment by grahamwhaley** [comment]
> thanks @bergwolf
> 
> nit: 80 character per line
> 
> CI is not happy
> 
> ```
> ERROR: Document VMT/KCSA/KCSA-CVE-2020-2024.md is not referenced
> ```

You need to index them into [this table](https://github.com/kata-containers/community/blob/master/VMT/KCSA.md) @bergwolf :-)

File: VMT/KCSA/KCSA-CVE-2020-2025.md (hunk 1 above)
Comment (amshinde) [reply VMT/KCSA/KCSA-CVE-2020-2025.md:28]: This probably needs to be https://github.com/kata-containers/runtime/pull/2487 instead @bergwolf

File: VMT/KCSA/KCSA-CVE-2020-2025.md [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+
+  type: GitHub
+
+reproduce:
+  - Create a Kata container using Cloud Hypervisor
+  - Create a new file on the guest rootfs
+  - View the guest rootfs image on the host and the file can be seen there
+
+notes:
+  - The vulnerability can be used to attack other guests by malicious containers that find other ways to gain control over the guest. And all users running Kata Containers on top of Cloud Hypervisor are recomended to upgrade.
```
Comment (amshinde) [inline VMT/KCSA/KCSA-CVE-2020-2025.md:43]: /recomended/recommended/

File: VMT/KCSA/KCSA-CVE-2020-2025.md (hunk 1 above)
Comment (bergwolf) [reply VMT/KCSA/KCSA-CVE-2020-2025.md:28]: The PR link is below in reviews section. This one is the issue link, although it doesn't talk much about the vulnerability.

**Comment by bergwolf** [comment]
updated. PTAL @amshinde @jodh-intel @devimc @grahamwhaley

File: VMT/KCSA/KCSA-CVE-2020-2025.md [hunk 3]
```diff
@@ -0,0 +1,62 @@
...
+
+notes:
+  - The vulnerability can be used to attack other guests by malicious containers
+    that find other ways to gain control over the guest. And all users running
+    Kata Containers on top of Cloud Hypervisor are recommended to upgrade.
+
+    When running Kata Containers with Cloud Hypervisor, any change made to root
+    filesystem device is written to the underlying .img file. Since the device
+    is plugged as read-write, a malicious guest could write to it through
+    utilities like debugfs.
```
Comment (YuvalAvra) [inline VMT/KCSA/KCSA-CVE-2020-2025.md:54]: I believe this is a mistake from my original advisory - `debugfs` isn't needed if the guest is malicious, the guest can simply create/modify files and the changes will propogate to the image file on the host.

Comment (devimc) [reply VMT/KCSA/KCSA-CVE-2020-2025.md:54]: @YuvalAvra  `malicious guest` is a container that found a way to escape from the namespaces in the guest?

Comment (YuvalAvra) [reply VMT/KCSA/KCSA-CVE-2020-2025.md:54]: That's how I refer to a guest running malicious code outside the container scope, but perhaps that's not Kata's terminology. It's not necessarily a container that escaped, for example `CVE-2020-2025` would result in subsequent guests being malicious without their containers escaping.

My point here is to separate distinct issues: `CVE-2020-2025` is exploitable from outside the container, if the container has a way to access the guest - that's a different issue.

Comment (bergwolf) [reply VMT/KCSA/KCSA-CVE-2020-2025.md]: Good point! I've re-phrased it and dropped `debugfs`.

File: VMT/KCSA/KCSA-CVE-2020-2024.md [hunk 4]
```diff
@@ -0,0 +1,68 @@
...
+    version: Before v1.11.0
+
+vulnerabilities:
+
+  - cve-id: CVE-2020-2024
+
+reporters:
+
+  - name: Yuval Avrahami
+    affiliation: Yuval Avrahami
```
Comment (bergwolf) [inline VMT/KCSA/KCSA-CVE-2020-2024.md:24]: hi @YuvalAvra, which affiliation would you like to be linked to? Generally it can be a company or organization.

Comment (YuvalAvra) [reply VMT/KCSA/KCSA-CVE-2020-2024.md:24]: Palo Alto Networks, thanks

**Comment by jodh-intel** [comment]
@YuvalAvra - would you like to do a final check before this is merged?

**Comment by jodh-intel** [comment]
Thanks @YuvalAvra.

**Pull request merged**